.vib/fluent-bit/goss/goss.yaml

```diff
@@ -0,0 +1,33 @@
+http://fluent-bit:{{ .Vars.service.ports.http }}/:
+  status: 200
+  body:
+    - /version.*{{ .Env.APP_VERSION }}/
+file:
+  /opt/bitnami/fluent-bit/conf:
+    exists: true
+    filetype: directory
+    mode: "0755"
```

Comment:
Remove the permissions check, as they have proven to be a source of false positives. If the path is K8s-related, the permissions may vary depending on the underlying cluster.

Reply:
[d884759](https://github.com/bitnami/charts/pull/15453/commits/d88475922fa9ce7829c72589eed76210a1ea9089)